examples/mxnet_imagenet_resnet50.py

```diff
@@ -317,6 +308,14 @@ def evaluate(epoch):
     if params is not None:
         hvd.broadcast_parameters(params, root_rank=0)
 
+    # Create optimizer
+    optimizer_params = {'wd': args.wd,
+                        'momentum': args.momentum,
+                        'lr_scheduler': lr_sched}
```

Comment:
Does `rescale_grad` default to `1.0 / batch_size` for gluon but not module? Maybe it would be good to set it here anyway for consistency, or document the difference somehow.

Reply:
This is a good question. Short answer is yes. For Gluon, we take care of it in [Trainer.step()](https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/gluon/trainer.py#L309) function so there is no need to set it in optimizer_params. But for Module, because we create optimizer first and wrap it with DistributedOptimizer, we need to specify [rescale_grad](https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/optimizer/optimizer.py#L51) in optimizer_params when creating optimizer.

I can add a comment here to document the difference.